CHANGELOG.md

```diff
@@ -4,6 +4,48 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
 
 ---
 
+## 2.13.0
+
+
+### New Features
+- Datastreams Monitoring (DSM): Adds support for schema tracking.
+- Exception Replay will capture any exceptions that are manually attached to a span with a call to `set_exc_info`.
+- LLM Observability: The LangChain integration now submits vectorstore `similarity_search` spans to LLM Observability as retrieval spans.
```

Comment:
These are probably ordered chronologically, but maybe we should start ordering notes by component so that they are grouped together rather than all over the place. WDYT?

Reply:
Yeah I like this idea a lot. I'll bring it up in the guild meeting this week.

Reply:
+ Maybe alphabetize?